apps/web-giddh/src/app/vat-report/obligations/obligations.component.html

```diff
@@ -98,8 +98,18 @@
                         <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
                     </table>
                     <section *ngIf="!tableDataSource?.length">
-                        <div class="no-data">
-                            <h1>{{ commonLocaleData?.app_no_data_found }}</h1>
+                        <div *ngIf="getFormControl('taxNumber').value">
+                            <div *ngIf="connectToHMRCUrl" class="initial-screen position-absolute text-center">
```

Comment:
why using position absolute ?

Reply:
Check use of text center if not required remove it.

Reply:
remove text-center DONE.
{{ localeData?.connect_to_hmrc }} in center  ,  use of position absolute.